src/openrct2/object/RideObject.cpp

```diff
@@ -218,6 +218,7 @@ void RideObject::Load()
             if (vehicleEntry->car_visual != VEHICLE_VISUAL_RIVER_RAPIDS)
             {
                 const auto numRotationFrames = vehicleEntry->GetNumRotationFrames();
+                vehicleEntry->NumRotationFrames = numRotationFrames;
```

Comment:
Why are you doing this?

Reply:
Instead of what?

Reply:
Well its not being used anywhere (write only) so seems odd.

Reply:
It got a little ahead of itself in the PR chain, it gets used a little in Cleanup Vehicle Drawing Code part 3 and a lot in Cleanup Vehicle Drawing Code Part 4

Reply:
okay thats fine. Just wanted to make sure it wasn't a mistake.